src/AppBundle/Entity/Base/PostalAddress.php

```diff
@@ -50,15 +51,27 @@
      * @Groups({"place"})
      * @ORM\Column(nullable=true)
      * @Assert\Type(type="string")
+     * @Assert\NotBlank(groups={"delivery_address"})
      * @ApiProperty(iri="https://schema.org/name")
      */
     private $name;
 
     /**
+     * @var string Family name. In the U.S., the last name of an Person. This can be used along with givenName instead of the name property.
+     *
+     * @ORM\Column(nullable=true)
+     * @Assert\Type(type="string")
+     * @Assert\NotBlank(groups={"delivery_address"})
+     * @ApiProperty(iri="https://schema.org/familyName")
+     */
+    private $familyName;
```

Comment:
Name is important IMHO. For ex. : you deliver to one person at Acme Company, you enter the building and need to find the person. (+ I think people are expecting us to take their name).

Arf I didn't get that `name` was for the address name.